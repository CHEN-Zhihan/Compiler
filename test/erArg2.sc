def main(a, a) {
    return a + a;
}

main(5, 6);