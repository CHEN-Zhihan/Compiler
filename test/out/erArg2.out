duplicate argument 'a' in function definition
